### ISSUE TITLE
Alignment bug?

### ISSUE THREAD

**Issue by camjc** [yannickcr/eslint-plugin-react#1246]
Hey, I think this should be picked up but its not:

```
  return (
    <P>
      <Clickable inNewTab to={guideUrls[productType]}>
        <Icon className={styles.icon} name="document" size="medium" />Product guide (PDF)
    </Clickable>
    </P>
  );
```
I figure `</Clickable>` should have six spaces behind it.

**Comment by jseminck** [comment]
Confirmed it does not complain about this being incorrect. Luckily, it also does not give any warnings/errors when the code is formatted correctly. :)

Will investigate!
